routers/voice.py

```
@@ -225,8 +226,7 @@ async def batch(request: Request, token: str = Form(...), session_id: str = Form
         tokens_record = await tokens_collections.find_one({"token": token, "is_active": 1})
 
         if tokens_record:
-            now = datetime.now()
-            date_time = now.strftime("%d/%m/%Y %H:%M:%S")
+            date_time = current_time()
```

Ruff (0.8.2)

Local variable `current_time` referenced before assignment

(F823)